learn/self_hosted/configure_meilisearch_at_launch.mdx

```diff
@@ -238,6 +238,13 @@ Meilisearch automatically collects data from all instances that do not opt out u
 
 [Read more about our policy on data collection](/learn/resources/telemetry), or take a look at [the comprehensive list of all data points we collect](/learn/resources/telemetry#exhaustive-list-of-all-collected-data).
 
+### Contact email
+
+**Environment variable**: `MEILI_CONTACT_EMAIL`<br />
+**CLI option**: `--contact-email`
```

Comment:
It also directly accepts the email (or any text). I don't know if it's clear enough 🤷‍♂️